index.html

```diff
@@ -2,4 +2,10 @@
 layout: default
 ---
 
-Hello world!
+<h1>Hello world</h1>
+
+<p class="intro">Deployments so easy it's almost magical</p>
```

Comment:
There is no `.intro` in Vanilla you could use the [heading class](https://docs.vanillaframework.io/en/patterns/headings).